### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by anna-shulgina**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -3,8 +3,82 @@
...
+
+  test('should return 2 for dog if age is >= 24 and < 29', () => {
+    expect(getHumanAge(1, 24))
+      .toEqual([0, 2]);
+  });
+
+  test(`should return [NaN, NaN] if type of variables equal 'string'`, () => {
+    expect(getHumanAge('one', 'two'))
+      .toEqual([NaN, NaN]);
+  });
```
Comment (oablokhina) [inline src/getHumanAge.test.js:83]: add more checks using boundary values and equivalence classes techniques

Comment (oablokhina) [reply src/getHumanAge.test.js:83]: don't add extra files to your PR in the future

**Review by oablokhina — CHANGES_REQUESTED** [review]

**Comment by anna-shulgina** [comment]
058f2267ef5dea34ad2e66606239192f1c313f44

**Review by nikitaminchenko — APPROVED** [review]
Changes from 2 commit were added, good job

**Review by oablokhina — APPROVED** [review]
